tests/foreman/installer/test_installer.py

```diff
@@ -1390,6 +1391,40 @@ def test_installer_options_and_sections(filter):
     assert previous == current, msg
 
 
+@pytest.mark.e2e
+@pytest.mark.tier1
```

Comment:
I think this can be run as part of `build_sanity` after snap-templatization, @jyejare wdyt?

Reply:
Basicaly, build sanity is something I wanted to test before snap templatization. We could run installer tests as first thing as thats the first pre-requisite of the build acceptance that build sanity helps us in !